TESTS/mbedtls/selftest/main.cpp

```diff
@@ -20,7 +20,6 @@
 #include "utest.h"
 #include "rtos.h"
 
-using namespace utest::v1;
```

Comment:
same as previous comment.
I would have moved this statement to line 51, instead of removing it, after all the header file includes. In this way, this specific .c file would still be using the namespace, but it won't affect all the header files.
The issue was that this statement was before the header files